person.rb

```diff
@@ -0,0 +1,35 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
+  attr_accessor :name, :age
```

Comment:
We need a rental to be listed on the setter/getter here because we need to read when we want to see who got the book, and also write it so that we read it.

Reply:
Done, as advised